ui/e2e/settings/index.spec.ts

```diff
@@ -110,6 +110,31 @@ test("it is possible to create and delete secrets", async ({ page }) => {
   ).toBeHidden();
 });
 
+test("secrets are displayed in alphabetical order", async ({ page }) => {
+  await page.goto(`/n/${namespace}/settings`);
+
+  // Firt Create secrets in non-alphabetical order
+  const secretNames = ["X", "T", "A", "01"];
+
+  for (const name of secretNames) {
+    await page.getByTestId("secret-create").click();
+    await page.getByPlaceholder("secret-name").type(name);
+    await page.locator("textarea").type("test-value");
+    await page.getByRole("button", { name: "Create" }).click();
+    await waitForSuccessToast(page);
+  }
+
+  // Then get all secret names in the list
+  const secretElements = page
+    .getByTestId("secrets-section")
+    .getByTestId("item-name");
+  const displayedNames = await secretElements.allTextContents();
+
+  // And check if the names are in alphabetical order
+  const sortedNames = [...secretNames].sort();
+  expect(displayedNames).toEqual(sortedNames);
```

Comment:
I have a small suggestion here. I would rather prefer to make a very explicit assertion without doing `sort()`. So like e.g. `expect(displayedNames).toEqual(["1", "2", "3", "..."]);`

I think it is easier to read (I was asking myself if the 01 comes first, but I would have to run the code to actual see) and it removes any (possibly wrong) assumption on how sorting works in JavaScript. I'm overdoing it a bit, but theoretically, you could have used the same function for sorting in your implementation and in your test, and you would have introduced the same bug in both.